docs/documentation/server_admin/topics/sso-protocols/con-oidc-auth-flows.adoc

```diff
@@ -89,7 +89,7 @@ It is possible to specify that the refresh token is considered invalid once it i
 which were already used, would not be considered valid anymore by {project_name}. This is possible to set with the use of _Revoke Refresh token_ option as specified in the <<_timeouts,timeouts section>>.
 
 {project_name} also supports the situation that no refresh token rotation exists. In this case, a refresh token is returned during login, but subsequent responses from refresh-token requests will not
-return new refresh tokens. This practice is recommended for instance in the link:{adapterguide_link}#_fapi-support[FAPI 2 draft specification].
+return new refresh tokens. This practice is recommended for instance in the *FAPI 2 draft specification* in the link:https://www.keycloak.org/guides#securing-apps[securing apps] section.
```

Comment:
Same like my other comment (can be hopefully attribute instead of hardcoded link).